src/OpenApi/Generate.elm

```diff
@@ -1025,20 +1085,25 @@ toRequestFunctions server effectTypes method pathUrl operation =
 
                 lamderaProgramTestCommands :
                     AuthorizationInfo
+                    -> List (Elm.Expression -> ( Elm.Expression, Elm.Expression, Bool ))
                     -> Elm.Annotation.Annotation
                     -> (Elm.Expression -> PerPackage Elm.Expression)
                     -> (Elm.Expression -> Elm.Expression)
                     -> ({ requireToMsg : Bool } -> PerPackage Elm.Annotation.Annotation)
                     -> List ( Common.Module, Elm.Declaration )
-                lamderaProgramTestCommands auth _ toBody replaced paramType =
+                lamderaProgramTestCommands auth toHeaderParams _ toBody replaced paramType =
                     declarationGroup auth
                         (\_ ->
                             { cmdArg =
                                 \config ->
                                     Elm.record
                                         [ ( "url", replaced config )
                                         , ( "method", Elm.string method )
-                                        , ( "headers", headersFromList Gen.Effect.Http.call_.header auth config )
+                                        , ( "headers"
+                                          , toHeaderParams
+                                                |> List.map (\f -> f config)
+                                                |> headersFromList Gen.Http.call_.header auth config
+                                          )
```

Comment:
I guess this repetition is not possible to fold into `headerFromList`?

Reply:
Oh, it might be? I'll double check